### ISSUE TITLE
Accessibility: get.gov choose your domain screen reader output

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/get.gov#314]
### Problem
**Input field**
get.gov's Start your .gov domain search programmatic label is `Check Domain Name input Check Domain input`
This fails section 508 success criteria (SC) 1.3.1-programmatic-label because the ANDI / screen reader output does not adequately define the form element.
The screen reader output should be:
`Start your .gov domain search See if the .gov domain you want is available. We'll work with you to find the best domain for your organization that meets our requirements. [Read more about choosing your .gov domain.](https://get.gov/domains/choosing/). Do not include .gov in your search.`

![Image](https://github.com/user-attachments/assets/5b580abe-a74a-4700-84e7-8d91de0f5207)

**Button**
the `Check availability` button screen reader output is `Check availability of Domain Name Check Domain Availability`. This fails section 508 success criteria (SC) 1.3.1-programmatic-label because the ANDI / screen reader output does not adequately define the form element.
The screen reader output should be:
`Start your .gov domain search Check domain availability `

![Image](https://github.com/user-attachments/assets/5e73c1ef-8a5c-4ebb-9574-7dd8d623913d)

**Comment by AnnaGingle** [comment]
This is a dev ticket
